starter_code/css/tablet.css

```diff
@@ -0,0 +1,23 @@
+img{
+  max-width: 100%;
+    height: auto;
+}
```

Comment:
This ruleset for the `img` selector isn't be applied universally since it is not contained w/ a media query. I recommend moving it to your base style sheet.